src/style.scss

```diff
@@ -1,6 +1,6 @@
 /* nunito-sans-regular - latin */
 @font-face {
-    font-family: 'Nunito Sans';
+    font-family: 'nunito-sans-regular-400 - latin';
```

Comment:
boyle bi font family yok. bide bu neden degistirildi?

Reply:
bu kodlar auto generate edildi
https://github.com/sumsets/web3-profile/pull/4
^ pr'in aciklamasina bi daha bakin pls